CppExport/src/exporter/CppExporter.cpp

```diff
@@ -34,74 +34,157 @@
 #include "Export/src/writer/FragmentLayouter.h"
 #include "ModelBase/src/model/TreeManager.h"
 
-#include "CodeUnit.h"
-#include "CodeComposite.h"
+#include "../CodeUnit.h"
+#include "../CodeComposite.h"
+#include "../Config.h"
 
 namespace CppExport {
 
 QList<ExportError> CppExporter::exportTree(Model::TreeManager* manager, const QString& pathToProjectContainerDirectory)
 {
-	auto project = DCast<OOModel::Project>(manager->root());
-	Q_ASSERT(project);
+	auto us = units(manager);
+	auto cs = mergeUnits(us);
 
-	auto codeComposite = new CodeComposite("Composite");
-	for (auto i = 0; i < project->classes()->size(); i++)
+	calculateSourceFragments(cs);
+	calculateDependencies(cs);
+
+	auto dir = new Export::SourceDir(nullptr, pathToProjectContainerDirectory + "/src");
+	for (auto c : cs) exportCodeComposite(c, dir);
+	auto layout = layouter();
+	Export::Exporter::exportToFileSystem("", dir, &layout);
+
+	return {};
+}
+
+QList<CodeUnit*> CppExporter::units(Model::TreeManager* treeManager)
```

Comment:
Do we need to have two `units` method? Is the other method called anywhere else but below? If so (or if you expect this to happen), where and why? If not, then please merge these into one method.